app/code/Magento/Store/Console/Command/StoreListCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Magento\Store\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+
+/**
+ * Class StoreListCommand
+ * @package Magento\Store\Console\Command
```

Comment:
phpdoc updated in commit a9ca0045d50d5a94e0f83fe7c9ef293e7957438e 

The format for this updated doc is in a format consistent with [app/code/Magento/Cron/Console/Command/CronCommand.php](https://github.com/magento/magento2/blob/develop/app/code/Magento/Cron/Console/Command/CronCommand.php#L21)